openedx/core/djangoapps/appsembler/sites/tasks.py

```diff
@@ -40,8 +40,7 @@ def import_course_on_site_creation_apply_async(organization):
     """
     return import_course_on_site_creation.apply_async(
         kwargs={'organization_id': organization.id},
-        retry=False,  # The task is not expected to be able to recover after a failure.
-        queue="edx.core.cms.high"
+        retry=False  # The task is not expected to be able to recover after a failure.
```

Comment:
No need to remove the comma.

```suggestion
        retry=False,  # The task is not expected to be able to recover after a failure.
```